_layouts/default.html

```diff
@@ -63,6 +63,54 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
```

Comment:
```suggestion
            "name": "Bisq Network",
```

Full name...also better differentiates from new `author` name "Bisq" (see below).